app/localoperator/DieterichRuinaAgeing.h

```diff
@@ -86,37 +86,54 @@ class DieterichRuinaAgeing {
         double snAbs = -sn + p_[index].get<SnPre>();
         double tauAbs = norm(tauAbsVec);
         double V = 0.0;
-        if (eta == 0.0) {
+        double max_xi = -std::pow(2, 8);
```

Comment:
For consistency with code you added further down, it would make sense that this is
```
int max_xi = 8
```

Reply:
done